salt/modules/state.py

```diff
@@ -1243,13 +1243,21 @@ def sls_id(
 
     The state ID comes *before* the module ID(s) on the command line.
 
+    id
+        ID to call
+
+    mods
+        single or coma separated list of modules where to find and resolve ID and it's requisites
```

Comment:
This might read more cleanly with something like:

`Comma-delimited list of modules to search for given id and its requisites`